study-builder/studies/testboston/snippets/question-followup-sample-collect-time-am-pm.conf

```diff
@@ -0,0 +1,60 @@
+{
+  include required("../../snippets/picklist-question-single-list.conf"),
+  "stableId": ${id.q.followup_sample_collect_time_am_pm},
+  "promptTemplate": {
+    "templateType": "HTML",
+    "templateText": "$followup_sample_collect_time_am_pm_prompt"
+    "variables": [
+      {
+        "name": "followup_sample_collect_time_am_pm_prompt",
+        "translations": [
+          { "language": "en", "text": "" },
+          { "language": "es", "text": "" },
+          { "language": "ht", "text": "" }
+        ]
+      }
+    ]
+  },
+  "picklistOptions": [
+    {
+      "stableId": "TIME_AM",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$sample_collect_time_am",
+        "variables": [
+          {
+            "name": "sample_collect_time_am",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.option.am} },
+              { "language": "es", "text": ${i18n.es.option.am} },
+              { "language": "ht", "text": ${i18n.ht.option.am} }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "stableId": "TIME_PM",
+      "optionLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "$sample_collect_time_pm",
+        "variables": [
+          {
+            "name": "sample_collect_time_pm",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.option.pm} },
+              { "language": "es", "text": ${i18n.es.option.pm} },
+              { "language": "ht", "text": ${i18n.ht.option.pm} }
+            ]
+          }
+        ]
+      }
+    },
+  ],
+  "validations": [
+    {
+      "ruleType": "REQUIRED",
+      "allowSave": true,
+    }
```

Comment:
I have noticed that these required rules are making it so you can submit the form without filling in a time, even if you didn't have to fill in date or symptoms... Same case in Baseline Symptom survey. Not sure if that's what we want.

Reply:
Date and Time (hour & Min) were made required, hence this is also required. Will double check and probably make all optional.

Reply:
confirmed with jen, its required.

Reply:
It doesn't look like the date part is marked required. I can fix that once this is merged.